frontend/src/views/Options.vue

```diff
@@ -303,6 +303,12 @@ export default Vue.extend({
     async setCurrentChain() {
       localStorage.setItem('currentChain', this.currentChain);
       Events.$emit('setting:currentChain', { value: this.currentChain });
+      (this as any).$router.replace(
```

Comment:
There's a router being used in common.ts without `as any` conversion. You could create and export a function in common.ts that will do `router.replace` and call it from here.

Reply:
Checked it, should work, you can also change (this as any).$router that's in NftDisplay component

Reply:
@kcper @Chriserus Thanks, worked!